lib/ansible/modules/network/netconf/netconf_config.py

```diff
@@ -9,112 +9,156 @@
+      it continues to process configuration data on erro, error is recorded and negative response is generated
+      if any errors occur. If value is C(rollback-on-error) it rollback to the original configuration in case
+      any error occurs, this requires the remote Netconf server to support the :rollback-on-error capability.
+    default: stop-on-error
+    choices: ['stop-on-error', 'continue-on-error', 'rollback-on-error']
+    version_added: "2.7"
   save:
     description:
-      - The C(save) argument instructs the module to save the running-
-        config to the startup-config if changed.
-    required: false
+      - The C(save) argument instructs the module to save the running-config to the startup-config if changed.
     default: false
     version_added: "2.4"
-  username:
+  backup:
     description:
-     - the username to authenticate with
-    required: true
-  password:
+      - This argument will cause the module to create a full backup of
+        the current C(running-config) from the remote device before any
+        changes are made.  The backup file is written to the C(backup)
+        folder in the playbook root directory or role root directory, if
+        playbook is part of an ansible role. If the directory does not exist,
+        it is created.
+    type: bool
+    default: 'no'
+    version_added: "2.7"
+  delete:
     description:
-     - password of the user to authenticate with
-    required: true
-  xml:
+      - It instructs the module to delete the configuration from value mentioned in C(target) datastore.
+    type: bool
+    default: 'no'
+    version_added: "2.7"
+  commit:
     description:
-     - the XML content to send to the device
-    required: false
-  src:
+      - This boolean flag controls if the configuration changes should be committed or not after editing the
+        candidate datastore. This oprion is supported only if remote Netconf server supports :candidate
+        capability. If the value is set to I(False) commit won't be issued after edit-config operation
+        and user needs to handle commit or discard-changes explicitly.
+    type: bool
+    default: True
+    version_added: "2.7"
+  validate:
     description:
-      - Specifies the source path to the xml file that contains the configuration
-        or configuration template to load.  The path to the source file can
-        either be the full path on the Ansible control host or a relative
-        path from the playbook or role root directory.  This argument is mutually
-        exclusive with I(xml).
-    required: false
-    version_added: "2.4"
-
-
+      - This boolean flag if set validates the content of datastore given in C(target) option.
+        For this option to work remote Netconf server shoule support :validate capability.
+    type: bool
+    default: False
+    version_added: "2.7"
 requirements:
   - "python >= 2.6"
```

Comment:
Is this something we still need to specify? We dropped support for python < 2.6 several releases ago now.

Reply:
Ansible 2.7 dropped support for Py2.6